packages/alfa-selenium/src/selenium.ts

```diff
@@ -0,0 +1,30 @@
+/// <reference lib="dom" />
+/// <reference types="node" />
+
+import * as device from "@siteimprove/alfa-device/native";
+import * as dom from "@siteimprove/alfa-dom/native";
+import { Device } from "@siteimprove/alfa-device";
+import { Document } from "@siteimprove/alfa-dom";
+import { Request, Response } from "@siteimprove/alfa-http";
+import { Page } from "@siteimprove/alfa-web";
+
+import { WebDriver } from "selenium-webdriver";
+
+/**
+ * @public
+ */
+export namespace Selenium {
+  export async function toPage(driver: WebDriver): Promise<Page> {
+    const deviceJSON = await driver.executeScript(device.Native.fromWindow);
+    const alfaDevice = Device.from(deviceJSON as Device.JSON);
+
+    const documentJSON = await driver.executeScript(dom.Native.fromNode);
+
+    return Page.of(
+      Request.empty(),
+      Response.empty(),
```

Comment:
Can we get the request and response more similar to how the web extension does it?

https://github.com/Siteimprove/alfa-web-extension/blob/5d2aff7e2f7539d473d3a1da5a6e22bdae53f498/src/api/actions/serialize/serialize-page.ts#L30-L41

I think this is missing from the other wrappers as well, so maybe it they all need to be fixed together in a separate PR.

Reply:
Yes, we'll handle it with II-8316.